quiz/classes/TextQuestion.class.ts

```diff
@@ -0,0 +1,22 @@
+import { Question } from './Question.class';
+import { Printer } from '../interfaces/Printer.interface';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+class TextQuestion extends Question {
+  maxLength: number;
+
+  constructor(private printer: Printer, maxLength, question, answer) {
+    super(question, answer);
+    this.maxLength = maxLength;
+  }
+
+  validation() {
+    if (this.maxLength <= 16) {
```

Comment:
Would be more real life-like, if it would validate `this.answer > this.maxLength` and return boolean that means is the answer is valid or not